docs/proposals/device-crd.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Device CRD Design
+authors:
+    - "@rohitsardesai83"
+approvers:
+  - "@qizha"
+  - "@CindyXing"
+  - "@kevin-wangzefeng"
+  - "@Baoqiang-Zhang"
+  - "@m1093782566"
+creation-date: 2019-02-27
+last-updated: 2019-02-27
+status: In Progress
+---
+
+## Motivation
+
+Currently kubeedge has an edge part which manages the lifecycle of applications
+deployed on the edge node, but there is no way to manage devices from the cloud.
```

Comment:
I wouldn't put this two lines in the design doc. Device management was in our original roadmap.

Reply:
removed the two lines. thanks.